tests/modules/test_detectionlists.py

```diff
@@ -75,7 +75,17 @@ def test_update_user_notes_calls_user_client_with_expected_values(
             TEST_USER_ID, "newnotes"
         )
 
-    def test_add_user_risk_tag_calls_user_client_with_expected_values(
+    def test_update_user_notes_returns_response(
+        self, mock_microservice_client_factory, mock_detection_list_user_client
+    ):
+        mock_microservice_client_factory.get_detection_list_user_client.return_value = (
+            mock_detection_list_user_client
+        )
+        module = DetectionListsModule(mock_microservice_client_factory)
+        response = module.update_user_notes(TEST_USER_ID, "newnotes")
+        assert response
```

Comment:
type is MagicMock in this case, but before was None, so these tests catch the error